api/namex/resources/requests.py

```diff
@@ -289,6 +296,40 @@ def get(*args, **kwargs):
             q = q.filter(RequestDAO.lastUpdate > text(
                 '(now() at time zone \'utc\') - INTERVAL \'{hour_offset} HOURS\''.format(hour_offset=current_hour + 24 * 29)))
 
+        if submittedInterval and (submittedStartDate or submittedEndDate):
+            return jsonify({"message": "submittedInterval cannot be used in conjuction with submittedStartDate and submittedEndDateTime"}), 400
```

Comment:
Also "submittedEndDateTime" -> "submittedEndDate".

Reply:
Fixed this.  Thanks